### PR TITLE
HOPSWORKS-967

### PR DESCRIPTION
**Pull request by o-alex**
adding hops/download_url as public attribute (metadata.rb) to point to hops artefacts download locations.

### CODE REVIEW

**Comment by o-alex** [comment]
so the general tests pass https://github.com/logicalclocks/hops-testing/pull/312

I tested on a local vm with using the new attributes.

File: attributes/default.rb [hunk 1]
```diff
@@ -45,8 +45,8 @@
 default['hops']['hdfs']['blocksize']           = "134217728"
 default['hops']['hdfs']['umask']               = "0022"
 
-default['hops']['url']['primary']              = node['download_url'] + "/hops-" + node['hops']['version'] + ".tgz"
-default['hops']['url']['secondary']            = "https://www.hops.site/hops-" + node['hops']['version'] + ".tgz"
+default['hops']['download_url']                = node['download_url']
+default['hops']['url']['primary']              = node['hops']['download_url'] + "/hops-" + node['hops']['version'] + ".tgz"
```
Comment (SirOibaf) [inline attributes/default.rb]: Generally the PR looks good. I'd just remove the `primary` and call the attribute something like `default['hops']['dist_url']` and change the name of the root url to something like `default['hops']['root_url']`. I suspect that with the current `download_url` people will get confused.

File: metadata.rb [hunk 2]
```diff
@@ -206,12 +206,8 @@
...
           :type => 'string'
 
-attribute "hops/url/primary",
-          :description => "Primary download url of hops distribution",
-          :type => 'string'
-
-attribute "hops/url/secondary",
-          :description => "Secondary download url of hops distribution",
+attribute "hops/download_url",
+          :description => "Download url of hops distribution artefacts",
```
Comment (SirOibaf) [inline metadata.rb]: typo

File: attributes/default.rb (hunk 1 above)
Comment (o-alex) [reply attributes/default.rb]: change addressed

File: metadata.rb (hunk 2 above)
Comment (o-alex) [reply metadata.rb]: change addressed

File: attributes/default.rb (hunk 1 above)
Comment (o-alex) [reply attributes/default.rb]: one more thing I need to do

**Comment by o-alex** [comment]
tests pass again 
https://github.com/logicalclocks/hops-testing/pull/312

**Pull request merged**